### ISSUE TITLE
is data_file_directories option available in scylla?

### ISSUE THREAD

**Issue by ban4785** [scylladb/scylla#2053]
*Installation details*
Scylla version : 1.5.0
Cluster size: 3
OS (CentOS 7.2):

*Hardware details (for performance issues)*          Delete if unneeded
Platform (physical):
Hardware: sockets= 32cores= hyperthreading= memory=64G
Disks: (SSD raid0)

in scylla.yaml file
i configured data_file_directories to several directory
example => - /var/lib/scylla/data1 - /var/lib/scylla/data2 - /var/lib/scylla/data3

after config, launched scylla server and load some data.
but only /var/lib/scylla/data1 directory only has data.
i found that cassandra 3.9 distributed data to several path, so cassandra 3.9 available this option.

is this value not supported in scylla?

i need this option because in raid0 with 4 ssd infra, 
one ssd fail cause raid0 disk unavailable. so i want to use several directories.

**Comment by infordb** [comment]
If there are 3 nodes and one is down, the repair effect is too great
During the repair period, the TPS decreased considerably.
![image](https://cloud.githubusercontent.com/assets/10610884/22581583/83734676-ea24-11e6-825b-7091f54baae9.png)
refer to  https://github.com/scylladb/scylla/issues/1907

So, we need to a way to reduce the impact of repair

**Comment by dorlaor** [comment]
You're asking to implement JBOD support. We may have such a github issue
already.
JBOD has an advantage but also has disadvantages - it doesn't utilize all
drives in parallel
and one drive can become full while others aren't.

The repair effect is a bug and should be fixed regardless. @asias has code
that improves
it and we have a work in progress to provide a better algorithm instead of
the current merkle tree.

**Comment by dorlaor** [comment]
Closing, duplicate of #830
